server.js

```diff
@@ -1,5 +1,24 @@
-const app = require('./app')
+const app = require("./app");
+const mongoose = require("mongoose");
 
-app.listen(3000, () => {
-  console.log("Server running. Use our API on port: 3000")
-})
+require("dotenv").config();
+const { DB_HOST: urlDb } = process.env;
+
+const connection = mongoose.connect(urlDb);
+
+const startServer = async () => {
+  try {
+    await connection;
```

Comment:
```suggestion
    await mongoose.connect(urlDb);
```

Reply:
To powyżej to sugestia, mam wrażenie że poprzez przypisywanie do connect to trochę więcej kodu niż powinno być, więcej trzeba śledzić. A connection jest użyty tylko raz. Dlatego napisałem powyższe.